src/Kunstmaan/NodeSearchBundle/Configuration/NodePagesConfiguration.php

```diff
@@ -318,7 +326,7 @@ public function setAnalysis(Index $index, AnalysisFactoryInterface $analysis)
      *
      * @return Mapping
      */
-    protected function createDefaultSearchFieldsMapping(Index $index, $lang = 'en')
+    protected function createDefaultSearchFieldsMapping(Index $index)
```

Comment:
Can you undo this change? I know it looks stupid but this is necessary to avoid BC breaks on older projects that still require $lang in this method.